bot/seasons/easter/april_fools_vids.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+
+class AprilFoolVideos(commands.Cog):
+    """A cog for april fools that gets a random april fools video from youtube."""
+    def __init__(self, bot):
+        self.bot = bot
+        self.yt_vids = self.load_json()
+        self.youtubers = ['google']  # will add more in future
+
+    @staticmethod
+    def load_json():
+        """A function to load json data."""
+        p = Path('bot/resources/april_fools_vids.json')
+        with p.open() as json_file:
+            all_vids = load(json_file)
+        return all_vids
+
+    @commands.command(name='fool')
+    async def aprial_fools(self, ctx):
+        """Gets a random april fools video from youtube."""
+        random_youtuber = random.choice(self.youtubers)
+        category = self.yt_vids[random_youtuber]
+        random_vid = random.choice(category)
+        embed = discord.Embed()
+        embed.title = random_vid['title']
+        embed.colour = Colours.yellow
+        embed.description = f'Checkout this april fools video by {random_youtuber}'
+        embed.url = random_vid['link']
+        await ctx.send(embed=embed)
```

Comment:
This doesn't need to be sent as two separate messages.

Reply:
how do i send it in 1 message?

Reply:
i think ill leave it there cuz i want the embed to be sent first.

Reply:
There should not be 2 messages sent for the same invocation. I don't see what the custom embed adds that can't be done by having the bot send the same "Check out this video" string along with a link to the video in the same message.

![image](https://user-images.githubusercontent.com/5323929/55414865-fb5a3280-5539-11e9-85b7-b0812d7b21c4.png)

Reply:
gotcha, im just too attracted to embeds, lol